packages/compass-saved-aggregations-queries/src/stores/copy-to-clipboard.tsx

```diff
@@ -0,0 +1,70 @@
+import type { RootActions, RootState } from './index';
```

Comment:
Should this file be `tsx`?

Reply:
Oh yeah, doesn't need to, will fix, thanks

Reply:
Done in https://github.com/mongodb-js/compass/pull/2785/commits/2f8e1b726627781ce3947b6803892a8680050fd1